frontend/src/metabase/query_builder/components/template_tags/SnippetSidebar/SnippetSidebar.jsx

```diff
@@ -164,70 +161,46 @@ class SnippetSidebarInner extends React.Component {
                     f(this, { className: CS.mr2 }),
                   ),
                 ]}
-                {snippets.length >= MIN_SNIPPETS_FOR_SEARCH && (
-                  <Icon
-                    className={cx(S.SearchSnippetIcon, {
-                      [S.isHidden]: showSearch,
-                    })}
-                    name="search"
-                    size={HEADER_ICON_SIZE}
-                    onClick={this.showSearch}
-                  />
-                )}
+                {snippets.length >= MIN_SNIPPETS_FOR_SEARCH &&
```

Comment:
The seach input is cut off. In this screenshots the search query is "jj", but first "j" looks like "i"

![image](https://github.com/user-attachments/assets/9e39ef0c-a81c-4b22-b929-d9fef5f1cd7c)